carla/data/helper/processing.py

```diff
@@ -0,0 +1,21 @@
+from sklearn import preprocessing
+
+
+def normalize(df, num_cols, scaler=None):
```

Comment:
Is this still a leftover from the rename (and back) to the `helper` package? I see there is the same one in `processing/processing.py`

Reply:
Yes, I missed that package while merging with main. Will delete this in the next commit